client/src/components/userprofiles/UserProfileDetails.jsx

```diff
@@ -14,6 +15,10 @@ export default function UserProfileDetails() {
   if (!userProfile) {
     return null;
   }
+
+  const saveImgClicked = () => {
+    updateImg(userProfile.id, image).then(() => {getProfileWithRolesById(id).then(setUserProfile)})
+  }
```

Comment:
Add a line of whitespace between 21 and 22, so that there is whitespace above the return